server/service/osquery_utils/queries.go

```diff
@@ -754,7 +755,7 @@ func GetDetailQueries(ac *fleet.AppConfig) map[string]DetailQuery {
 		generatedMap["users"] = usersQuery
 	}
 
-	if ac != nil && ac.HostSettings.EnableScheduledQueryStats {
+	if ac != nil && ac.HostSettings.EnableScheduledQueryStats && fleetConfig.App.EnableScheduledQueryStats {
```

Comment:
Should we just have this only on `config.FleetConfig`? (Otherwise it's error-prone/confusing to have the option in app and config.FleetConfig...)

Reply:
We could, sure. @zwass thoughts?

Reply:
Well, I went ahead and remove it from app config.